sentry/client_test.go

```diff
@@ -84,3 +85,63 @@ func TestErrFrames(t *testing.T) {
 	err := ex.New("this is only a test")
 	assert.NotEmpty(errFrames(err))
 }
+
+func TestErrEventFingerprintDefault(t *testing.T) {
+	assert := assert.New(t)
+
+	event := errEvent(context.Background(), logger.ErrorEvent{
+		Flag: logger.Fatal,
+		Err:  ex.New("this is a test", ex.OptMessage("a message")),
+		State: &http.Request{
+			Method: "POST",
+			Host:   "example.org",
+			TLS:    &tls.ConnectionState{},
+			URL:    webutil.MustParseURL("https://example.org/foo"),
+		},
+	})
+
+	assert.NotNil(event)
+	assert.NotEmpty(event.Fingerprint)
+	assert.Equal("this is a test", event.Fingerprint[0])
```

Comment:
Nit: I'd prefer

```suggestion
	assert.Equal([]string{"this is a test"}, event.Fingerprint)
```

as future-proofing against that case the `event.Fingerprint` has more than 1 element.